app/helpers/getBubbleContextType.tsx

```diff
@@ -0,0 +1,13 @@
+export const BUBBLE_CONTEXT_TYPE = "b_c_t";
+
+const store = require("store");
+
+export function setBubbleContextTypeHelper() {
+  let bubbleContextType: number = store.get(BUBBLE_CONTEXT_TYPE) || 1;
```

Comment:
people who read this code cannot understand what 1, 2, 3 means.
use `enum` or at least `CONSTANT` for a comprehensible code.